src/types/types.ts

```diff
@@ -0,0 +1,35 @@
+export type PostType = {
+  id: number
+  message: string
+  likeCount: number
+}
+
+export type ProfileType = {
+  userId: number
+  lookingForAJob: boolean
+  lookingForAJobDescription: string
+  fullName: string
+  contacts: ContactsType
+  photos: PhotosType
+}
```

Comment:
PhotosType[] ?

Reply:
as I understand it, JS allows ignoring this clarification